sdk/storage/Azure.Storage.Webjobs.Extensions.Blobs/tests/BlobConfigurationTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Storage.Queues;
+using Azure.WebJobs.Extensions.Storage.Common.Tests;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Host.FunctionalTests
+{
+    public class BlobConfigurationTests : IClassFixture<AzuriteFixture>
```

Comment:
Would be great to annotate class with `[Collection(AzuriteCollection.Name)]` (since that went through) instead of implementing `IClassFixture`. less azurite will be created. If you decide to do it, plz give `container/blob` more specific names to avoid conflicts (I wish we could use dynamic values with attributes).